public/gameInfo.php

```diff
@@ -1384,8 +1390,12 @@ function ResetProgress() {
                 if ($numAchievements == 0) {
                     echo "<li><a class='btn py-2 mb-2 block' href='/setRequestors.php?g=$gameID'><span class='icon icon-md ml-1 mr-3'>📜</span>Set Requestors</a></li>";
                 }
+                if ($numAchievements >= 1 && (!isset($guideURL) || (empty($guideURL)))) {
+                    echo "<a class='btn py-2 mb-2 block' href='https://github.com/RetroAchievements/guides/wiki' target='_blank' rel='noreferrer'><span class='icon icon-md ml-1 mr-3'>📖</span>Create a Guide</a>";
+                }
```

Comment:
I'm not sold on having a "Create" button on every page. Most users will never use it, and [#1352](https://github.com/RetroAchievements/RAWeb/pull/1352/files#diff-8e708c2e13ee510e7add867723475fd6944bcb7661604d8456d720e0f9c540a7R153) adds this same link to the Community menu.

Reply:
Fair enough, removed